boards/arm/bloodlight_rev2/bloodlight_rev2.dts

```diff
@@ -6,7 +6,7 @@
 
 /dts-v1/;
 #include <st/g4/stm32g474Xe.dtsi>
-#include <st/g4/stm32g474c(b-c-e)tx-pinctrl.dtsi>
+#include <st/g4/stm32g474r(b-c-e)tx-pinctrl.dtsi>
```

Comment:
The bloodlight's revivision board 2 has a stm32g474cb (mine is a stm32g474ce)on it according to https://github.com/CodethinkLabs/bloodlight-hardware/blob/master/hardware/plots/rev2/bloodlight.pdf. Why was then decided to use the devicetree declaration for a stm32g474r board?

Reply:
It was because in the late April version of Zephyr I was using, the original include was missing. I've just switched to Zephyr v2.6.0 and the original include works fine. So I have reverted this change and pushed. I also reverted the changes to the LED config which weren't in the main branch.